src/sagemaker/sklearn/estimator.py

```diff
@@ -37,10 +37,10 @@ class SKLearn(Framework):
     def __init__(
         self,
         entry_point,
-        framework_version=defaults.SKLEARN_VERSION,
+        framework_version=None,
```

Comment:
This is defaulting `framework_version` to `None` right? I think we need make it as a required parameter. Or do we have other considerations here?

Reply:
good question. the intention is to allow users to specify either `image_name`/`image` or `framework_version` (and `py_version`). we validate either `framework_version` and `py_version` are provided or `image_name` is provided.